Doc/source/designspaceLib/readme.rst

```diff
@@ -297,6 +297,8 @@ RuleDescriptor object
 -  Each condition is a dict with ``name``, ``minimum`` and ``maximum`` keys.
 -  ``subs``: list of substitutions
 -  Each substitution is stored as tuples of glyphnames, e.g. ("a", "a.alt").
+-  Note: By default, rules are applied *before* text shaping/OpenType layout. See 
+   `5.0 rules element`_.
```

Comment:
Sorry, I tried hard but reStructuredText is very hard to use. I can't figure out how to make this link go to &sect; 5.0 rules element but say "the `<rules>` Attributes section".

Reply:
it's ok, I don't know either :)